extension/src/json-viewer/highlighter.js

```diff
@@ -97,11 +97,18 @@ Highlighter.prototype = {
   },
 
   bindMousedown: function() {
+    var self = this;
+    console.log(self.openLinksInNewWindow())
```

Comment:
Hi, thanks for the PR. It's a great addition! Can you remove the `console.log` before I merge? Thanks!